src/sys/socket/mod.rs

```diff
@@ -803,6 +803,38 @@ pub enum ControlMessageOwned {
     #[cfg(any(target_os = "linux"))]
     TlsGetRecordType(TlsGetRecordType),
 
+    #[cfg(any(
+        apple_targets,
+        linux_android,
+        target_os = "freebsd",
+    ))]
+    #[cfg(feature = "net")]
+    IpTos(libc::c_int),
```

Comment:
Does this need to be `u8` on linux_android?

Reply:
I don't think so.

Reply:
Actually, hang on. The tests pass but my actual app is not.